x-pack/plugin/rollup/src/test/java/org/elasticsearch/xpack/rollup/job/RollupJobTaskTests.java

```diff
@@ -47,6 +47,7 @@
 
 public class RollupJobTaskTests extends ESTestCase {
 
+    private static final Settings SETTINGS = Settings.builder().put("node_name", "test").build();
```

Comment:
Isn't the name of this setting `node.name` instead of `node_name`?

Reply:
Checking!

Reply:
Yeah, I've got it wrong. I'll fix.